src/server/helpers/render.js

```diff
@@ -16,12 +16,46 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+// @flow
 
 import * as utils from './utils';
 import Handlebars from 'handlebars';
 
 
-function renderRelationship(relationship) {
+type EntityInRelationship = {
+	bbid: string,
+	defaultAlias?: {name: string},
+	type: string
+};
+
+type Relationship = {
+	source: EntityInRelationship,
+	target: EntityInRelationship,
+	type: {displayTemplate: string}
+};
+
+/**
+ * @typedef {Object} EntityInRelationship
+ * @property {string} bbid
+ * @property {?Object} defaultAlias
+ * @property {string} defaultAlias.name
+ * @property {string} type
+ */
+
+/**
+ * @typedef {Object} Relationship
+ * @property {EntityInRelationship} source
+ * @property {EntityInRelationship} target
+ * @property {Object} type
+ * @property {string} type.displayTemplate
+ */
+
+/**
+ * Returns the markup corresponding to the given entity relationship.
```

Comment:
Would be good to mention what this is used for (rendering relationships on entity display pages), and also what defines the markup (the display template).